libs/dh/metering-point/domain/src/lib/is-metering-point-id.spec.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { isMeteringPointId } from './is-metering-point-id';
+
+describe(isMeteringPointId.prototype.name, () => {
+  it('is valid', () => {
+    const _18Digits = '123456789000000000';
+
+    expect(isMeteringPointId(_18Digits)).toBeTruthy();
+  });
+
+  describe('NOT valid', () => {
+    it('empty string', () => {
```

Comment:
Consider writing your spec's like you're telling a story ex.
```suggestion
    it('should invalidate empty strings', () => {
```